package.json

```diff
@@ -21,6 +21,15 @@
   ],
   "main": "./out/extension.js",
   "contributes": {
+    "configuration":{
+      "properties": {
+        "justAntlr.useBuildCompileCache":{
```

Comment:
Added configuration for using build cache, allowing for free selection of whether to use uploaded build results